libs/perun/utils/src/lib/perun-utils.ts

```diff
@@ -864,3 +865,33 @@ export function isMemberIndirectString(member: RichMember): string {
   }
   return 'UNKNOWN';
 }
+
+export function escapeDangerousHtml(html: string): string {
+  const config = {
+    FORBID_TAGS: ['script', 'link', 'iframe', 'object', 'embed', 'frame', 'frameset', 'style'],
+  };
+  const sanitized = sanitize(html, config);
+
+  // Sanitize style properties (this is not supported by DOMPurify)
```

Comment:
This doesn't seem right. Skimmed through the DOMPurify docs and it seemed like there is a way to sanitize attribute values using the 'uponSanitizeAttribute' hook. (See [here](https://github.com/cure53/DOMPurify/tree/main/demos#hook-to-allow-and-sand-box-all-javascript-link) for inspiration). However if you tested your method thoroughly and it works, fine by me.

Reply:
I guess i did not see it there :+1: Probably will use this hook instead of my method.